test/node_smoke_tests/module/document_passed.mjs

```diff
@@ -0,0 +1,14 @@
+import { JSDOM } from "jsdom";
+
+import { ensureJQuery } from "./lib/ensure_jquery.mjs";
```

Comment:
Note that all of these use the `.mjs` extension. This could be avoided by adding a simple `package.json`:
```json
{
  "type": "module"
}
```
to the `test/node_smoke_tests/module/` folder. However, some of node smoke test runs actually test `exports` from within the package:
https://nodejs.org/api/packages.html#self-referencing-a-package-using-its-name
That way, we can easily test whether what we define via `exports` really works.

The problem is, this feature requires the closest `package.json` to the the module `package.json` so a root one in our case. This means we cannot add such a simple `package.json` to `test/node_smoke_tests/module/`.

These are just tests run in Node.js, nothing to be consumed externally, so I think it should be fine.